pm_muairss_read/pm_muairss_read.xml

```diff
@@ -0,0 +1,44 @@
+<tool id="pm_uep_opt_read" name="Cluster outputs (pm-muairs_read)" version="0.1.0" python_template_version="3.5">
+    <requirements>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        pip show pymuon-suite || ( git clone https://github.com/muon-spectroscopy-computational-project/pymuon-suite /tmp/pymuon-suite && pip install /tmp/pymuon-suite && rm -rf /tmp/pymuon-suite) &&
+        unzip $folder &&
+        out_folder="`python ${__tool_directory__}/get_out_folder.py`" &&
+        pm-muairss -t r input.cell params.yaml && 
+        zip -r out_zip.zip \$out_folder *_clusters* param.yaml input.cell &&
+        find \$out_folder > tree.txt
+    ]]></command>
+    <inputs>
+        <param type="data" name="folder" format="zip" />
+    </inputs>
+    <outputs>
+        <data label="cluster-out" name="out_zip" format="zip" from_work_dir="out_zip.zip"/>
+        <data label="clusters" name="out_cluster" format="txt" from_work_dir="$out_folder/*clusters.txt"/>
+        <data label="clusters_dat" name="out_cluster_dat" format="txt" from_work_dir="$out_folder/*clusters.dat"/>
+    </outputs>
+    <tests>
+        <test>
+            <param name="folder" value="uep-out.zip" ftype='zip'/>
+            <output name="out_zip" file="cluster-out.zip" ftype="zip" compare="sim_size" delta_frac="0.1"/>
+            <output name="out_cluster" file="clusters.txt" ftype="txt" compare="sim_size" delta_frac="0.01"/>
+            <output name="out_cluster_dat" file="clusters_dat.dat" ftype="txt" compare="sim_size" delta_frac="0.001"/>
+        </test>
+    </tests>
```

Comment:
Resolving as I think this would actually be a pymuonsuite issue rather than a Galaxy issue if it didn't work. (since it doesn't require any local dependencies like the DFTB+ stuff)